conans/test/unittests/client/tools/cppstd_required_test.py

```diff
@@ -0,0 +1,192 @@
+import unittest
+from mock import mock
+from parameterized import parameterized
+
+from conans.test.utils.conanfile import MockConanfile, MockSettings
+from conans.client.tools import OSInfo
+from conans.errors import ConanInvalidConfiguration
+
+from conans.tools import check_min_cppstd, valid_min_cppstd
+
+
+class UserProneTests(unittest.TestCase):
```

Comment:
UserProneTests? What does it mean?

Reply:
Renamed to UserInputTests. The idea here is testing possible wrong entries.